### DISCUSSION TITLE
Jest not recognize type of macro function

### DISCUSSION THREAD

**Discussion by DATeclipse** [lingui/js-lingui#1269] [Q&A]
Does anyone know of a solution to get our Jest to recognize the type for the Macro function of lingui's `t`? Our current setup was working with `react-script` and our original build w/ `create-react-app` but we've transitioned to NextJS and replaced our test command with `jest --watch` and it's completely failing at this level.

<img width="728" alt="Screenshot 2022-07-06 at 12 49 01" src="https://user-images.githubusercontent.com/12127546/177543633-5e77cd3a-dbaf-4383-9e31-9ca18b3b50b9.png">

Any guidance or direction would help.

**Comment by timofei-iatsenko** [comment]
Latest nextjs jest runner uses swc instead of babel. So you need to directly override transpiler in jest config to use babel instead of what nextjs provides.
